src/einsteinpy/coordinates/utils.py

```diff
@@ -6,6 +6,22 @@
 _c = constant.c.value
 
 
+class BaseError(Exception):
+    def __init__(self, *args, **kwargs):
+        self.message = f"{' '.join(args)}"
```

Comment:
You don't need to use f-strings or any string formatting here, since it's just the contents of `args`.

Although, this would raise a `TypeError`, if `args` contains non-strings. Consider `self.message = ' '.join(str(i) for i in args)`